### DISCUSSION TITLE
Is there a way to view QField on your desktop?

### DISCUSSION THREAD

**Discussion by katemunnik** [opengisch/QField#4801] [Q&A]
Hello - I love the QFIELD concept and it will work really well with our team, especially non-GIS experts. However, it would be better for these non-GIS experts to make their edits on their desktop / field laptops rather than a phone/tablet. 
Is there a possibility to do this?
I have tried using an Android emulator, but the required panning functionality that you need to make edits n digitizing mode doesn't work on the emulator I used. 
Any ideas?

**Comment by VxTedxV** [comment]
On [releases page](https://github.com/opengisch/qfield/releases), try windows or macOS (2.8.7 or below) assets.
